package.json

```diff
@@ -75,13 +75,16 @@
     "husky": "^7.0.4",
     "merkletreejs": "^0.2.31",
     "prettier": "^2.3.2",
-    "prettier-plugin-solidity": "^1.1.1",
+    "prettier-plugin-solidity": "^1.3.0",
     "release-it": "^15.0.0",
     "solhint": "^3.3.7",
     "solidity-coverage": "^0.7.21",
     "solmate": "^6.6.1",
     "ts-node": "^10.1.0",
     "typechain": "^5.1.2",
     "typescript": "^4.5.2"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^5.0.2"
```

Comment:
you don't need this anymore do you?

Reply:
I think we need this for the MockERC20 code

Reply:
ok, but can you move this to devDependencies and also replace solmate with openzeppelin?

We currently use solmate for MockERC20

https://github.com/LooksRare/contracts-libs/blob/master/test/mock/MockERC20.sol

Reply:
Would you also like me to replace solmate with openzeppelin for MockERC721, MockERC1155 and WETH (in LowLevelWETH.t.sol)?

Reply:
Yes do it pls

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/0ba5d3ac227b9d0b200461e3cf70d5bfbe69cfa4